public/app/features/dashboard-scene/sharing/ExportButton/ExportMenu.tsx

```diff
@@ -50,6 +50,15 @@ export default function ExportMenu({ dashboard }: { dashboard: DashboardScene })
       onClick: () => onMenuItemClick(shareDashboardType.export),
     });
 
+    menuItems.push({
+      shareId: shareDashboardType.image,
+      testId: newExportButtonSelector.exportAsImage,
+      icon: 'camera',
+      label: t('share-dashboard.menu.export-image-title', 'Export as Image'),
+      renderCondition: Boolean(config.featureToggles.dashboardImageSharing),
```

Comment:
should we also disable this menu option if image renderer is not enabled / configured?

Reply:
This is the current behavior of "Export PDF" (showing "Export as PDF" in the menu and showing a warning in the drawer) and I think it's fine. It allows the user to know why the option is disabled.